mandatory/9-choose-your-own-adventure.js

```diff
@@ -0,0 +1,135 @@
+/*
+Create a "Choose Your Own Adventure" game using an object. In these kind of
+games, the player is in a room and can move to other rooms to the north, east,
+south or west.
+
+To start the game, run this file with Node as usual. To stop the game, press
+Ctrl-C.
+
+It has a currentRoom property to store which room the player is in.
+
+Give your object methods for:
+
+- Starting the game in the correct room when passed a room name parameter
+- Moving the player to another room when passed a direction parameter
+
+The functionality for running a game has been provided for you. It first prompts
+the player to enter the starting room. Then it will ask players to to type in a
+direction (north/east/south/west) that they want to move in.
+
+A room object has been provided for you. Take your time to read it carefully.
```

Comment:
```suggestion
A rooms object has been provided for you. Take your time to read it carefully.
```

Or "An object containing one object per room" or similar?

Reply:
Thanks, fixed in 108b7de